### ISSUE TITLE
Currency filter is missing option to control sign position and dot/comma setting

### ISSUE THREAD

**Issue by telamon** [freearhey/vue2-filters#31]
Hi!
I checked the source for the currency filter and realized that there is no way to specify the position of the
currency sign or specify if delimiters should be dots or commas. 

In some locales like Swedish the rules for currency presentation is different e.g.
given `123456789 | currency('kr')`
The filter produces:
`kr1,234,567.89`
But according to Swedish standard the formatting should be:
`1 234 567,89 kr`

In other words, I'd like to submit a feature-request to be able to control the decimal-separator and the thousand-separator and of course placement of currency-sign.

**Comment by freearhey** [comment]
Closed due inactivity.

**Comment by marcushultman** [comment]
Opened pull request https://github.com/freearhey/vue2-filters/pull/41.
